### PR TITLE
creating article editor and recipe editor roles

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: config/sync/workflows.workflow.content_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - node.type.recipe
+  module:
+    - content_moderation
+id: content_workflow
+label: 'Content Workflow'
+type: content_moderation
+type_settings:
+  states:
+    draft:
+      label: Draft
```
Comment (ElXyfer) [inline config/sync/workflows.workflow.content_workflow.yml]: From the looks of this you reused the state of draft, you should remove this and create a new state called unpublished as per the task in notes.

File: config/sync/user.role.content_administrator.yml [hunk 2]
```diff
@@ -41,6 +41,8 @@ permissions:
   - 'edit site setting entities'
   - 'edit terms in categories'
   - 'edit terms in tags'
+  - 'use content_workflow transition create_new_draft'
+  - 'use content_workflow transition publish'
```
Comment (ElXyfer) [inline config/sync/user.role.content_administrator.yml]: Set these permissions for the administrator, we're going to be moving away from the "content administrator" role

File: config/sync/workflows.workflow.content_workflow.yml (hunk 1 above)
Comment (ElXyfer) [reply config/sync/workflows.workflow.content_workflow.yml]: Pull request should not have any instances of draft.

File: config/sync/user.role.content_administrator.yml (hunk 2 above)
Comment (katlazarova) [reply config/sync/user.role.content_administrator.yml]: done

File: config/sync/workflows.workflow.content_workflow.yml (hunk 1 above)
Comment (katlazarova) [reply config/sync/workflows.workflow.content_workflow.yml]: done

**Pull request merged**